web/services/wallet_services.py

```
@@ -0,0 +1,26 @@
+        mnemonic_phrase = mnemo.generate(strength=128)
+        # Convert the mnemonic to a seed
+        seed = mnemo.to_seed(mnemonic_phrase)
+        # Generate a wallet from the seed
+        account = Account.from_key(seed[:32])
+        return {"mnemonic": mnemonic_phrase, "address": account.address, "private_key": account.key.hex()}
+    except Exception as e:
+        # Log the error
+        import logging
+
+        logger = logging.getLogger(__name__)
+        logger.error(f"Error creating wallet: {e!s}")
```

Ruff (0.8.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)